src/main/java/org/opensearch/ad/rest/handler/ModelValidationActionHandler.java

```diff
@@ -252,7 +278,10 @@ private void getBucketAggregates(
         Map<String, Object> topEntity
     ) throws IOException {
         List<String> featureFields = ParseUtils.getFeatureFieldNames(anomalyDetector, xContentRegistry);
-        AggregationBuilder aggregation = getBucketAggregation(latestTime);
+        AggregationBuilder aggregation = getBucketAggregation(
+            latestTime,
+            (IntervalTimeConfiguration) anomalyDetector.getDetectionInterval()
```

Comment:
Suggest to remove all these downcasts to `IntervalTimeConfiguration` and just change `AnomalyDetector`s interval field from `TimeConfiguration` -> `IntervalTimeConfiguration`

Reply:
Will do this in separate PR as it creates dozens of failing test that I don't want to prioritize right now along with making sure there was no specific reason why we had `TimeConfiguration` used and not `IntervalTimeConfiguration` in the first place (seems to be parsing related probably)

Reply:
Sure, sounds good